src/modules/awake/Awake/Core/APIHelper.cs

```diff
@@ -63,10 +83,21 @@ public class APIHelper
 
         static APIHelper()
         {
+            _timedLoopTimer = new System.Timers.Timer();
             _log = LogManager.GetCurrentClassLogger();
             _tokenSource = new CancellationTokenSource();
         }
 
+        public static void SetConsoleControlHandler(ConsoleEventHandler handler, bool addHandler)
+        {
+            SetConsoleCtrlHandler(handler, addHandler);
+        }
+
+        public static void DeallocateConsole()
```

Comment:
It's not used anywhere

Reply:
Good catch - I was using it previously, and will remove it.